region/norfair/east/Frog Speedway.json

```diff
@@ -534,6 +534,40 @@
         "Safely kill the Beetoms from inside the speed blocks."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Shot Block Overload (Speedless Speedway)",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "Wave",
+        {"or": [
+          {"and": [
+            "Ice",
+            "Morph"
+          ]},
+          {"ammo": {"type": "Missile", "count": 6}},
+          {"ammo": {"type": "Super", "count": 6}},
+          "ScrewAttack",
+          "h_canBombThings",
+          {"enemyDamage": {
+            "enemy": "Beetom",
+            "type": "contact",
+            "hits": 15
+          }}
+        ]}
+      ],
+      "reusableRoomwideNotable": "Frog Speedway Shot Block Overload (Speedless Speedway)",
+      "note": [
+        "Shoot diagonal upward shots into the ceiling while slowly moving to hit all of the shot blocks and overload PLMs.",
+        "Because the shot blocks do not break, Samus needs to shoot over a larger region than when not in G-Mode - starting from around a half-screen away from the speed blocks.",
```

Comment:
I just stood in the normal spot and shot wave beams until it worked.

Reply:
Hmm, I can't get that to work. do you press against the speed blocks and stand and shoot up left?

Reply:
Yeah, hold left while shooting diagonally up-left.  Works in direct or indirect gmode.

Reply:
Weird, I tried a few times. Not sure what I was doing wrong.